CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 * [ENHANCEMENT] Ruler: Add more detailed query information to ruler query stats logging. #1411
 * [ENHANCEMENT] Admin: Admin API now has some styling. #1482 #1549
 * [ENHANCEMENT] Alertmanager: added `insight=true` field to alertmanager dispatch logs. #1379
+* [ENHANCEMENT] Store-gateway: Add the experimental ability to run index header operations in a dedicated thread pool. #1660
```

Comment:
Please mention this feature in the list of experimental ones at `docs/sources/operators-guide/configuring/about-versioning.md`.

Reply:
In this CHANGELOG entry, please also mention the new config option introduced.

Reply:
Fixed in 9df3e28966c5f6fd56f397af04358c616d7b522b